### PR TITLE
AU Core Server & Client CapabilityStatement - add missing interactions (https://github.com/hl7au/au-fhir-core/issues/121)

### PR DESCRIPTION
**Pull request by dbojicic**
Fixes https://github.com/hl7au/au-fhir-core/issues/121. 

Add missing interactions to AU Core Client.

### CODE REVIEW

File: input/resources/au-core-client.xml [hunk 1]
```diff
@@ -494,6 +494,9 @@
 <li>
 <strong>SHALL</strong> support <code>read</code>,<code>search-type</code>.
 </li>
```
Comment (dt-r) [inline input/resources/au-core-client.xml:496]: This change only does the narrative. Also need to add to the underlying resources. MAY that is in the narrative needs to be on each resource with a conformance obligation extension of MAY.

**Review by dt-r — CHANGES_REQUESTED** [review]
See comment.

That comment will also apply to AU Core Server.

**Comment by dbojicic** [comment]
@dt-r , updated AU Core Server to add conformance obligation extension of MAY - ready for review.

File: input/resources/au-core-server.xml [hunk 2]
```diff
@@ -3674,17 +3794,53 @@
...
-<referencePolicy value="resolves">
-<extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
-<valueCode value="SHOULD"/>
-</extension>
-</referencePolicy>
-<searchRevInclude value="Provenance:target">
+<interaction>
 <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
-<valueCode value="SHALL"/>
+<valueCode value="MAY"/>
```
Comment (dt-r) [inline input/resources/au-core-server.xml:3799]: Provenance:target is still SHALL. Revert this change.

File: input/resources/au-core-server.xml [hunk 3]
```diff
@@ -4488,8 +4924,7 @@
 <definition value="http://hl7.org/fhir/SearchParameter/Practitioner-name"/>
 <type value="string"/>
 </searchParam>
-</resource>
-<!-- PRACTITIONERROLE :: DONE -->
+</resource><!-- PRACTITIONERROLE :: DONE -->
```
Comment (dt-r) [inline input/resources/au-core-server.xml:4927]: Missing interactions for PractitionerRole

**Review by dt-r — CHANGES_REQUESTED** [review]
2 changes needed on AU Core Server.

File: input/resources/au-core-server.xml (hunk 3 above)
Comment (dbojicic) [reply input/resources/au-core-server.xml:4927]: Added

File: input/resources/au-core-server.xml (hunk 2 above)
Comment (dbojicic) [reply input/resources/au-core-server.xml:3799]: I cannot find Provenance:target as MAY anywhere?!

**Review by dt-r — APPROVED** [review]
